### PR TITLE
Added postgres minibroker brain test

### PR DESCRIPTION
**Pull request by f0rmiga**
## Description

Brain test for minibroker postgres.

## Test plan

```
$ (export FISSILE_ROLES=acceptance-tests-brain ; make scf-release && fissile build packages && fissile build images && make/kube)
$ make/tests acceptance-tests-brain env.INCLUDE="postgres"
```

### CODE REVIEW

File: make/tests [hunk 1]
```diff
@@ -92,7 +92,7 @@ do
     echo 1>&2 "${POD_NAME} container failed to reach Running state"
     exit 1
   fi
-  sleep 1
+  sleep 5
```
Comment (f0rmiga) [inline make/tests]: @mook-as It's not a random change :)

Comment (mook-as) [reply make/tests]: Great! Why is it here? :D

File: src/scf-release/src/acceptance-tests-brain/test-resources/postgres-example-app/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Postgres example app
+
+This is an example app initially intended to be used by the Postgres minibroker brain tests.
```
Comment (mook-as) [inline src/scf-release/src/acceptance-tests-brain/test-resources/postgres-example-app/README.md]: Do we not have an existing sample app that can talk postgres?

Comment (mook-as) [reply src/scf-release/src/acceptance-tests-brain/test-resources/postgres-example-app/README.md]: Now that I think about it… we have https://github.com/scf-samples/rails-example. Why the heck didn't I use that one for mariadb? I'm dumb. Sorry.

File: make/tests (hunk 1 above)
Comment (f0rmiga) [reply make/tests]: Because 1 second it too fast. I noticed sometimes it failed before the pod was ready.

Comment (mook-as) [reply make/tests]: :+1: (I just wanted _some_ sort of reason to show that it's not random :stuck_out_tongue:)

Comment (f0rmiga) [reply make/tests]: I didn't run that beautiful sed command in this file this time.

**Pull request merged**